ci/jenkins/test.sh

```diff
@@ -280,13 +282,15 @@ function deliver_antrea_windows {
         done
 
         # Use a script to run antrea agent in windows Network Policy cases
-        if [ "$TESTCASE" == "windows-networkpolicy" ]; then
+        if [ "$TESTCASE" == "windows-networkpolicy-temp-disabled" ]; then
```

Comment:
what do you mean about "windows-networkpolicy-temp-disabled"? what about "windows-networkpolicy-process"?

Reply:
Updated.